packages/styles/stories/Components/calendar/legend.example.html

```diff
@@ -0,0 +1,275 @@
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--legend-3">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">24</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--weekend fd-calendar__item--legend-9">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">25</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--weekend">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">26</span>
+                    </div>
+                </td>
+            </tr>
+            <tr class="fd-calendar__row">
+                <td role="gridcell" class="fd-calendar__item">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">27</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">28</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">1</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">2</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">3</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">4</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">5</span>
+                    </div>
+                </td>
+            </tr>
+            <tr class="fd-calendar__row--legend">
```

Comment:
you can't have a modifier class without the base class that you modify. `fd-calendar__row--legend` says it modifies `fd-calendar__row` base class, but there's no base class.
The correct way would be `fd-calendar__row fd-calendar__row--legend`
`fd-calendar__row` would bring the base CSS code and `fd-calendar__row--legend` would bring the CSS that is specific for the legend case